src/spox/_scope.py

```diff
@@ -114,18 +122,20 @@ def __delitem__(self, item: Union[str, H]):
         del self.of_name[key]
         del self.name_of[value]
 
-    def enum(self, base: str, suffix: str = "_{}") -> str:
+    def enum(self, base: str) -> str:
         """Find an unused name by enumerating the pattern ``base + suffix.format(i)`` through `i = 0, 1, ...`"""
```

Comment:
```suggestion
        """Find an unused name by enumerating the pattern ``f"{base}_{i}"`` through `i = 0, 1, ...`"""
```
This internal docstring is out of date now - something like this?